src/components/App.js

```diff
@@ -64,7 +65,11 @@ export class App extends Component {
             <MuiThemeProvider theme={createMuiTheme(theme)}>
               <AuthenticationSender />
               <AccessTokenSender />
-              <WorkspaceArea />
+              <Suspense
+                fallback={<div />}
```

Comment:
Right now we just show an empty div when loading. But we could put a spinner or other type of stuff here.